compiler/circle-part-eval/parts/Part_Add_Sqrt_000.part

```diff
@@ -0,0 +1,7 @@
+[partition]
+backends=cpu,acl_cl
+default=cpu
+comply=opcode
+
+[OPCODE]
+SQRT=acl_cl
```

Comment:
There's no `Add`. Is it right?

Reply:
`ADD`, as not assigned explicity here in `OPCODE` section, then it's default: `cpu` is assigned.

Reply:
Or if my answer is not what you asked for, can you elaborate your question?

Reply:
> ADD, as not assigned explicity here in OPCODE section, then it's default: cpu is assigned.

That exactly explains:)